rollup.config.js

```diff
@@ -0,0 +1,44 @@
+import resolve from '@rollup/plugin-node-resolve'
```

Comment:
Might want to look at something like https://github.com/pikapkg/pack for bundling/distribution. Will eliminate a lot of the boilerplate and deals with creating all the various versions + types that consumers might want to use.

At the very least, any modern lib should expose an ES module, which is what we would be using.

Reply:
I actually gave it a shot but was getting weird errors and found it hard to debug. It's possible that some other changes I made afterwards in this PR would have fixed it, but I found it personally easier to use rollup since I've used it before.

I agree thought that Pika looks like it would take the maintenance weight off if.. definitely feel free to submit a PR if you get it running!

Reply:
Also, incase it was unclear, currently we have an ES tree-shakeable build: https://bundlephobia.com/result?p=restii@0.2.2), as well as a CJS build if this were to be used in Node (which currently wouldn't work since we use DOM methods such as `Headers`, but will be helpful if we add SSR support)